### ISSUE TITLE
why jupyter notebook extension need install python extension

### ISSUE THREAD

**Issue by tjx666** [DonJayamanne/typescript-notebook#55]
I want to install jupyter notebook extension to support tool bar, but It require to install python extension. I don't write python, so I don't want to install python extension. Any idea about it?

**Comment by DonJayamanne** [comment]
I agree, I'm sorry that was an oversight, will look into an alternative.
